pkgs/applications/blockchains/dogecoin.nix

```
@@ -1,29 +1,32 @@
-{ lib, stdenv , fetchFromGitHub
-, pkg-config, autoreconfHook
-, db5, openssl, boost, zlib, miniupnpc, libevent
-, protobuf, util-linux, qt4, qrencode
-, withGui }:
+{ lib, stdenv, fetchFromGitHub, pkg-config, autoreconfHook, db5
+, openssl, boost, zlib, miniupnpc, libevent, protobuf, qtbase ? null
+, wrapQtAppsHook ? null, qttools ? null, qmake ? null, qrencode, withGui
+, hexdump, Cocoa ? null }:
```

```suggestion
, wrapQtAppsHook, qttools, qmake, qrencode, withGui
, hexdump, Cocoa }:
```

That should be required if it is behind a with flag.